### ISSUE TITLE
RGB: Backend

### ISSUE THREAD

**Issue by CalrethilOfMirkwood** [CalrethilOfMirkwood/flask_portfolio#13]
Develop the backend of the RGB project.  Use Pillow Image to put messages inside of the images

**Comment by CalrethilOfMirkwood** [comment]
I completed this task last night:
commit - https://github.com/CalrethilOfMirkwood/flask_portfolio/commit/473b341bbfdeb9068086d493325f4ec44f042695
file - [https://github.com/CalrethilOfMirkwood/flask_portfolio/blob/main/image.py](url)

Since the size of lassen-volcano-256.jpg is dramatically larger than the square pngs, I added a new value to the dictionary img_list so that the font on each image could be sized appropriately.

Even though I got everything to work, I ran into a critical error, the path of the images could not be found in just /static/img/(imagename).  I found a temporary workaround by fully specifying the local path of the images.  It seems almost everyone with a Windows computer is having this same issue, so Mr. Mortensen is going to look into a better solution.
